src/Nethermind/Nethermind.Evm/VirtualMachine.cs

```diff
@@ -181,6 +183,37 @@ public CallResult(ReadOnlyMemory<byte> output, bool? precompileSuccess, bool sho
     public interface IIsTracing { }
     public readonly struct NotTracing : IIsTracing { }
     public readonly struct IsTracing : IIsTracing { }
+
+    internal sealed class CodeLruCache
+    {
+        private const int CacheCount = 16;
```

Comment:
why 16 and not Environment.ProcessorCount for example?

Reply:
Distribution based on data; `x0-xf` in the nibble, 16 values